<ui version="4.0" >
 <class>newDocument</class>
 <widget class="QWidget" name="newDocument" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="titleLine" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Authors</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="authorsLine" />
   </item>
   <item row="8" column="2" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLineEdit" name="tagsLine" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Document</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLineEdit" name="documentLine" />
   </item>
   <item row="5" column="2" >
    <widget class="QPushButton" name="browseButton" >
     <property name="text" >
      <string>&amp;Browse</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QPushButton" name="searchButton" >
     <property name="text" >
      <string>&amp;Search CiteSeerX</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QLineEdit" name="yearLine" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QLineEdit" name="doiLine" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>DOI</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleLine</tabstop>
  <tabstop>authorsLine</tabstop>
  <tabstop>tagsLine</tabstop>
  <tabstop>yearLine</tabstop>
  <tabstop>doiLine</tabstop>
  <tabstop>documentLine</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
